### DISCUSSION TITLE
Camera recording storage size

### DISCUSSION THREAD

**Discussion by Disincentivato** [blakeblackshear/frigate#14148] [General Support]
Hi,
I'm new to Frigate, I just set up my 3 cameras and I have a few questions that are still unanswered.
I'll start with the first one.
I noticed that one of my camera (esterna) consumes almost 5 times more space than the others, even though they are the same brand and configured at the same way.
**What could be the reason?**

I use a NUC miniPC + Home assistant + Coral accelerator

![IMG_7438](https://github.com/user-attachments/assets/e45f626e-cb38-4410-9afa-2ffd3e439cdf)

**Comment by NickM-27** [comment]
Camera compression works by reducing space used by the frame that is similar. If one camera has more motion than others the compression won't work as well

**Reply by Disincentivato** [reply]
Thanks, all clear, I thought there wasn't that much of a difference, surely the external camera is more prone to detecting movements.
Do you think I get improvements if I use the masks on hedges that moves?

**Reply by NickM-27** [reply]
Recording is on the camera so you'd have to apply the mask on the camera
